idris.cabal

```diff
@@ -252,7 +252,7 @@ Library
                 , Tools_idris
 
   Build-depends:  base >=4 && <5
-                , aeson >= 0.6 && < 1.1
+                , aeson >= 0.6 && < 1.0.2.2
```

Comment:
I am unsure if this is intentional, but you are actually restricting it more not widening it.